package.json

```diff
@@ -52,6 +52,7 @@
     "@types/lodash": "4.17.16",
     "@types/react": "18.3.12",
     "@types/react-dom": "18.3.1",
+    "@types/react-modal": "3.16.3",
```

Comment:
What additional functionality does this bring that we do not already have? If we need this, should we refactor our other modals to use it?

Reply:
[Line 5 in LibraryCardFormModal](https://github.com/DataBiosphere/duos-ui/pull/2883/files#diff-7df667067f81993f377c532ec08ecf4f0d4f728cc3d7adef5c95268945d057d0R4) requires this for typescript compilation. I'm sure that there are other usages of this but I didn't see any in TSX files yet, just JSX files.

Reply:
I have a `ModalWrapper.jsx` that allows us to use our existing modals with Typescript.